### DISCUSSION TITLE
Frigate+: what is the new blue bounding box?

### DISCUSSION THREAD

**Discussion by DaveTSG** [blakeblackshear/frigate#15423] [Ask A Question] [answered]
As seen in the screenshot provided, Frigate+ has started showing light blue bounding boxes when you view an image in the Images tab. Previously, users could draw bounding boxes in white, and that is still the case.

However, Frigate+ is now showing - for example this screenshot - two bounding boxes: one in white and one in light blue.

![Screenshot 2024-12-10 at 8 38 40 AM](https://github.com/user-attachments/assets/c498ae14-83a5-4868-8f27-48b56f564b1c)

What is the significance/meaning of the blue bounding box? There doesn't seem to be anything about it in [the documentation](https://docs.frigate.video/integrations/plus/#annotate-and-verify).

Do we need to delete one of the two bounding boxes around the same object?

Thank you.

**Comment by NickM-27** [comment, marked as answer]
It is a suggestion, you will want to delete it in this case as the object is already labeled. CC @blakeblackshear perhaps this should be de-duplicated?

**Reply by DaveTSG** [reply]
Ah great, thank you.

**Reply by DaveTSG** [reply]
Is there a way of explicitly accepting the suggestion, or just leave it as is and click the "save & verify" button?

**Reply by NickM-27** [reply]
>  or just leave it as is and click the "save & verify" button?

yes

**Reply by DaveTSG** [reply]
> > or just leave it as is and click the "save & verify" button?
> 
> yes

Thanks.

**Reply by blakeblackshear** [reply]
It was probably just barely under the threshold to be counted as a duplicate bounding box. I should be able to improve this further in the future.
